apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -52,7 +54,9 @@ const Auth: React.FC = () => {
 	};
 
 	if (hasUser) {
-		return <Navigate to={AppRoute.ROOT} />;
+		navigate(AppRoute.ROOT, { replace: true });
+
+		return;
```

Comment:
```suggestion
		return <Navigate replace to={AppRoute.ROOT} />
```
You can just add replace prop. Why do you need to use useNavigate?

Reply:
Yeah, missed it, thanks